cpp/src/plasma/store.cc

```diff
@@ -104,7 +104,7 @@ GetRequest::GetRequest(Client* client, const std::vector<ObjectID>& object_ids)
   num_objects_to_wait_for = unique_ids.size();
 }
 
-Client::Client(int fd) : fd(fd) {}
+Client::Client(int fd) : fd(fd), notification_fd(boost::none) {}
```

Comment:
Let's make notification_fd = -1 by default if not set (and get rid of the boost dependency)